book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Let's add a method for adding rentals to create the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`. ✍🏻 
- Your implementation might look similar to the following:

```ruby
   .
  def add_rental(.., date)
    ...
  end
```

- Note:  The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.